datalad/support/gitrepo.py

```diff
@@ -1794,7 +1793,7 @@ def commit(self, msg=None, options=None, _datalad_msg=False, careless=True,
 
         file_chunks = generate_file_chunks(files, cmd) if files else [[]]
 
-        runner = WitlessRunner(cwd=self.path, env=env)
+        runner = GitWitlessRunner(cwd=self.path, env=env)
```

Comment:
BTW, spotted this added in the recent commit in master.  IMHO it should also use `GitWitlessRunner` to use "correct" git etc.